cli/commands/quote.ts

```diff
@@ -87,26 +101,20 @@ export class Quote extends BaseCommand {
       }
     }
 
-    const chainId = ID_TO_CHAIN_ID(chainIdNumb);
-
-    const log = this.logger;
-    const tokenProvider = this.tokenProvider;
-    const router = this.router;
```

Comment:
These unused?

Reply:
I moved them up in the definition because I wanted to access the log before it was defined, let me revert this change because it's not necessary

Reply:
oh i see, missed it up there